### PR TITLE
added neumorphism toggle buttons

### PR DESCRIPTION
**Pull request by suyogkokaje**

### CODE REVIEW

**Comment by suyogkokaje** [comment]
Sorry for again making PR. But I faced some issue. So I closed previous PR and made again. Please have a look.

**Review by nandini7637 — CHANGES_REQUESTED** [review]
Kindly follow the folder pattern
Buttons->Toggle buttons_01->Code(includes html, css file)
                                                ->Readme

**Comment by suyogkokaje** [comment]
@nandini7637 please have a look.

**Review by bhavesh-chaudhari — APPROVED** [review]
Requested changes have been made. LGTM.

**Review by nandini7637 — APPROVED** [review]

File: Buttons/Toggle buttons_01/code/style.css [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    background-color: #d1dad3;
+    box-shadow: -3px -3px 5px rgba(255,255,255,0.5),
+                3px 3px 5px rgba(0,0,0,0.25);
+    transition: 0.3s ease-in-out;
+}
+
+.switch-toggle input[type="checkbox"]:checked+label::before{
+    left: 50%;
+    content: "ON";
+    color: #fff;
```
Comment (Abirpal202049) [inline Buttons/Toggle buttons_01/code/style.css:92]: Change this to #1f1f1f it think this will be better and visible

**Review by Abirpal202049 — CHANGES_REQUESTED** [review]

File: Buttons/Toggle buttons_01/code/style.css (hunk 1 above)
Comment (Abirpal202049) [inline Buttons/Toggle buttons_01/code/style.css:92]: Change the color from white to black for more visibility . I think #1f1f1f will be better

**Review by Abirpal202049 — CHANGES_REQUESTED** [review]

**Review by saikatmu31 — APPROVED** [review]

**Comment by suyogkokaje** [comment]
@Abirpal202049 please have a look.

**Review by Abirpal202049 — APPROVED** [review]

**Pull request merged**